src/firebase/utils/batchUpdateData.js

```diff
@@ -0,0 +1,57 @@
+import db from '../firebase';
+import { awardConsiderationEnum } from '../../constants';
+
+/*
+* This batchUpdateData method is written explicitly for handling
```

Comment:
Let's space this such that all the asterisks are in line with the asterisk on line 4.